lib/Store.js

```diff
@@ -28,6 +28,8 @@ var Store = (function () {
 
     var self = this;
     this.callback = callback;
+    this.pending = false;
```

Comment:
These will always be internal to Biff right? Should they be something like `__pending` so there aren't strange collisions?